101-azurefirewall-premium/azuredeploy.json

```diff
@@ -0,0 +1,256 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Deployment location"
+            }
+        }
+    },
+    "variables": {
+        "firewallAddressSpace": "10.0.100.0/24",
+        "firewallPrivateIPAddress": "10.0.100.4",
+        "serverAddressSpace": "10.0.10.0/24",
+        "keyVaultName": "[concat('fw-demo-', uniqueString(resourceGroup().id))]",
+        "keyVaultCASecretName": "CACert"
+    },
+    "resources": [
+        {
+            "apiVersion": "2020-05-01",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

Reply:
Thanks